File: 72151/src/main/java/com/uniminuto/biblioteca/entity/MovimientoInventario.java
package com.uniminuto.biblioteca.entity;

import java.io.Serializable;
import java.sql.Timestamp;
import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;
import javax.persistence.Table;
import lombok.Data;

@Data
@Entity
@Table(name = "MOVIMIENTOS_INVENTARIO")
public class MovimientoInventario implements Serializable {
    
    private static final long serialVersionUID = 1L;
    
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id_movimiento")
    private Integer idMovimiento;
    
    @ManyToOne
    @JoinColumn(name = "id_inventario", nullable = false)
    private Inventario inventario;
    
    @ManyToOne
    @JoinColumn(name = "id_usuario", nullable = false)
    private Usuario usuario;
    
    @ManyToOne
    @JoinColumn(name = "id_tipo_mov", nullable = false)
    private TipoMovimiento tipoMovimiento;
    
    @ManyToOne
    @JoinColumn(name = "id_direccion_origen")
    private Ubicacion ubicacionOrigen;
    
    @ManyToOne
    @JoinColumn(name = "id_direccion_destino")
    private Ubicacion ubicacionDestino;
    
    @Column(name = "cantidad_bultoa", nullable = false)
    private Integer cantidadBultoa;
    
    @Column(name = "fecha_movimiento")
    private Timestamp fechaMovimiento;
}

File: 72151-front/src/app/models/movimiento-inventario.ts
import { Inventario } from './inventario';
import { TipoMovimiento } from './tipo-movimiento';
import { Ubicacion } from './ubicacion';
import { Usuario } from './usuario';

export class MovimientoInventario {
    id_movimiento?: number;
    id_inventario: number;
    id_usuario: number;
    id_tipo_mov: number;
    id_direccion_origen?: number;
    id_direccion_destino?: number;
    cantidad_bultoa: number;
    fecha_movimiento?: Date;
    inventario?: Inventario;
    tipo_movimiento?: TipoMovimiento;
    usuario?: Usuario;
    ubicacion_origen?: Ubicacion;
    ubicacion_destino?: Ubicacion;
}

